src/main/java/io/antmedia/muxer/MuxAdaptor.java

```diff
@@ -2266,6 +2269,9 @@ else if(recordType == RecordType.WEBM) {
 			logger.error("Unrecognized record type: {}", recordType);
 		}
 
+		LocalDateTime ldt = LocalDateTime.now();
```

Comment:
Hi @LaCrak27 ,
Thank you for the fix and contribution

It may throw Nullpointer Exception, I'll just slightly change [the place of this lines](https://github.com/ant-media/Ant-Media-Server/blob/master/src/main/java/io/antmedia/muxer/Muxer.java#L734) to make it work in all cases



Regards